pkgs/servers/teleport/default.nix

```diff
@@ -3,6 +3,7 @@ let
   f = args: rec {
     teleport_15 = import ./15 args;
     teleport_16 = import ./16 args;
+    teleport_17 = import ./17 args;
     teleport = teleport_16;
```

Comment:
Let's make 17 the default, and add an entry to 25.05 release notes similar to this:

https://github.com/NixOS/nixpkgs/blob/695022dedc3d712e662acb73ad0f6b4daa284323/nixos/doc/manual/release-notes/rl-2411.section.md#L462-L464